syft/generic/frameworks/hook/hook_args.py

```diff
@@ -711,6 +711,7 @@ def register_tensor(
         response_ids: List of ids where the tensor should be stored
             and each id is pop out when needed.
     """
+    # owner.de_register_obj(tensor)
```

Comment:
That's not needed finally?

Reply:
Yes. Works without it.
Removed it as it was breaking the inplace operations such as __iadd__, etc

Reply:
If it's not needed, let's remove it.

Reply:
@karlhigley Done